include/fuzzy/ngram_matches.hh

```diff
@@ -9,17 +9,17 @@
 
 namespace fuzzy
 {
-  class Range {
-  public:
-    Range(size_t p_start,
-          size_t suffix_first, size_t suffix_last,
-          size_t match_length):_p_start(p_start), _suffix_first(suffix_first),
-                               _suffix_last(suffix_last), _match_length(match_length)
+  struct Range {
```

Comment:
Maybe we should remove this constructor ?

Reply:
It seems to be useful to initialize all attributes in one call. What do you think?

Reply:
It seems redundant with something like this `Range({suffix_first, suffix_last, match_length})`

Reply:
Right, this works too. I'll do that.